src/Misc/layoutbin/checkScripts/makeWebRequest.js

```diff
@@ -0,0 +1,75 @@
+const https = require('https')
+const http = require('http')
+const hostname = process.env['HOSTNAME']
+const port = process.env['PORT']
+const path = process.env['PATH']
+const pat = process.env['PAT']
+const proxyHost = process.env['PROXYHOST']
+const proxyPort = process.env['PROXYPORT']
+const proxyUsername = process.env['PROXYUSERNAME']
```

Comment:
you might want to coalesce with `''` so we dont get `null` when string concat below